Formula/kde-karchive.rb

```diff
@@ -0,0 +1,44 @@
+class KdeKarchive < Formula
+  desc "Reading, creating, and manipulating file archives"
+  homepage "https://api.kde.org/frameworks/karchive/html/index.html"
+  url "https://download.kde.org/stable/frameworks/5.71/karchive-5.71.0.tar.xz"
+  sha256 "cc81e856365dec2bcf3ec78aa01d42347ca390a2311ea12050f309dfbdb09624"
+  head "https://invent.kde.org/frameworks/karchive.git"
+
+  depends_on "cmake" => [:build, :test]
+  depends_on "doxygen" => :build
+  depends_on "graphviz" => :build
+  depends_on "kde-extra-cmake-modules" => [:build, :test]
+  uses_from_macos "bzip2"
+  depends_on "qt"
+  depends_on "xz"
+  uses_from_macos "zlib"
```

Comment:
Does this work with the style check? It seems a little easier to read.

```suggestion
  depends_on "doxygen" => :build
  depends_on "graphviz" => :build
  depends_on "cmake" => [:build, :test]
  depends_on "kde-extra-cmake-modules" => [:build, :test]
  depends_on "qt"
  depends_on "xz"
  
  uses_from_macos "bzip2"
  uses_from_macos "zlib"
```

Reply:
I had tried this, but `brew style --fix` wants cmake on top.